arch/arm/boot/dts/bcm283x.dtsi

```diff
@@ -337,6 +337,30 @@
 			status = "disabled";
 		};
 
+		i2c10mux: i2c10mux {
```

Comment:
`i2c0mux`?

Reply:
```bsc0mux``` if you wish.
i2c0 is a physical device, but this mux is also controlling /dev/i2c<something else> connected to the camera and display port.